assets/js/pages/SettingsPage/SettingsPage.jsx

```diff
@@ -149,8 +164,23 @@ function Settings() {
           </div>
         </div>
       </div>
+      <div className="py-4">
+        {softwareUpdatesSettingsLoading && (
+          <LoadingBox
+            className="shadow-none rounded-lg"
+            text="Loading Settings..."
+          />
+        )}
+        {!softwareUpdatesSettingsLoading && (
+          <SuseManagerConfig
+            url={settings.url}
+            username={settings.username}
+            certUploadDate={settings.ca_uploaded_at}
+          />
+        )}
+      </div>
```

Comment:
You could use a ternary here but no big deal

`softwareUpdatesSettingsLoading ? ... : ...`